pmlc/dialect/eltwise/ir/ops.td

```diff
@@ -56,6 +56,7 @@ def ScalarConstantOp :
 
   let extraClassDeclaration = [{
     Attribute getValue() { return getAttr("value"); }
+    void setValue(Attribute attr) {setAttr("value", attr); }
```

Comment:
I don't think we need to modify the value, we only need to modify the result type. The value itself can have whatever physical precision it has. But the logical type should follow `floatx` or `intx`. All constant ops basically have 2 types: physical (this is the type on the attribute) and a logical type (which type is returned from this op).